include/bigg.hpp

```diff
@@ -92,7 +90,5 @@ namespace bigg
 		uint32_t mWidth;
 		uint32_t mHeight;
 		const char* mTitle;
-		bool  mMousePressed[ 3 ];
```

Comment:
I utilize these states in a distributed tooling environment, perhaps support both: logical representation of an application; and, client renderer implementation. Picking a "source of truth" would be cool but starts to go out of scope :p 

otherwise, great work getting it in-line with the other imgui bootstrap strategies 👍

Reply:
The values in that array were only used to pass press events from the GLFW callback to the imguiEvents method as far as I can see. The value at a key's index should have only been true during the frame that key was pressed since all the values in the array were [reset to false after being used](https://github.com/JoshuaBrookover/bigg/pull/21/files/9aff1750f57d1937660927d43a88345393f13694#diff-07128d8a57c95ae7ccb0b4e508f8b741L167) every frame.

I could implement an isDown style array for mouse and keyboard buttons that hold the state of every button. Would that solve the problem?

Reply:
The mMouseWheel variable was also only used to pass callbacks to imguiEvents and [reset to 0 after being used](https://github.com/JoshuaBrookover/bigg/pull/21/files/9aff1750f57d1937660927d43a88345393f13694#diff-07128d8a57c95ae7ccb0b4e508f8b741L170) every frame.

I could store the sum of all scrollCallback events if that would be useful.